test/mnemonic-test.js

```diff
@@ -64,4 +67,127 @@ describe('Mnemonic', function() {
     assert.strictEqual(m2.language, m1.language);
     assert.bufferEqual(m2.toSeed(), m1.toSeed());
   });
+
+  it('should return true for isMnemonic when passed a Mnemonic object', () => {
+    const m1 = new Mnemonic();
+    assert.strictEqual(Mnemonic.isMnemonic(m1), true);
+  });
+
+  it('should return false for isMnemonic when passed a string', () => {
+    const m1 = new Mnemonic();
+    assert.strictEqual(Mnemonic.isMnemonic(m1.getPhrase()), false);
+  });
+
+  it('should return the phrase from toString', () => {
+    const m1 = new Mnemonic();
+    assert.strictEqual(m1.toString(), m1.getPhrase());
+  });
```

Comment:
What's interesting about the `Mnemonic` class is that it actually has no values right after construction (aka `new`). So in your original test and those that follow, `m1` doesn't even have a phrase until you try to compare it with `assert.strictEqual()` You can add these lines to your test to demonstrate what I mean:

```suggestion
  it('should return the phrase from toString', () => {
    const m1 = new Mnemonic();
    assert.strictEqual(m1.phrase, null);
    assert.strictEqual(m1.entropy, null);
    const phrase = m1.getPhrase();
    assert.notStrictEqual(m1.phrase, null);
    assert.notStrictEqual(m1.entropy, null);
    assert.strictEqual(m1.toString(), phrase);
  });
```